calico.go

```diff
@@ -313,8 +313,9 @@ func cmdAdd(args *skel.CmdArgs) error {
 					Labels: map[string]string{conf.Name: ""},
```

Comment:
I think we don't need these labels any more.  They don't help or hurt, so maybe we can just leave them as-is, but might be good to clean up.